scripts/xsslint/xsslint/default_config.py

```diff
@@ -0,0 +1,22 @@
+# Default xsslint config module.
```

Comment:
I'm not sure what the default config should look like. Maybe it should mirror the edx-platform-specific config for now so that people can continue to run the script without the extra option?